roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -132,14 +132,14 @@ defaults
   option redispatch
   option forwardfor
   option httpclose # needed for empty reponse body 204's
-  timeout client 300s
-  timeout server 300s
-  timeout connect 1s
-  timeout http-keep-alive 60s
-  timeout http-request 30s
+  timeout client {{ haproxy.defaults.timeout_client }}
+  timeout server {{ haproxy.defaults.timeout_server }}
+  timeout connect {{ haproxy.defaults.timeout_connect }}
+  timeout http-keep-alive {{ haproxy.defaults.timeout_http_keep_alive }}
+  timeout http-request {{ haproxy.defaults.timeoute_http_request }}
```

Comment:
misspelling, please change all 3 occurrences in the PR to "timeout_http_request"